docs/paper/admin/reference/configuration/bukkit-commands-configuration.mdx

```diff
@@ -0,0 +1,65 @@
+---
+slug: /reference/legacy-commands-configuration
+---
+
+# Bukkit Commands Configuration
+
+import React from 'react';
+import Config from '@site/src/components/Config';
+
+:::info
+
+The below YAML shows you the structure and default values for the legacy `commands.yml`.
+
+Click on a leaf node to view the description for that setting.
+
+:::
+
+:::warning
+
+The aliases defined in this legacy file do not support modern features
+- Tab completion
+- Permissions
+
+For these, see `config/command-aliases.yml` (Pending PR merge) {/*[`config/command-aliases.yml`](docs/paper/admin/reference/configuration/command-aliases.mdx)*/}. {/* TODO create other docs page */}
```

Comment:
Blocking on PaperMC/Paper#9971 being merged, and a docs page being added for this to link to.
@Machine-Maker should I add that page in this pr as well?

Reply:
I'd hold off, at least until someone else voices support for the format of that file. Don't want you doing a whole page only for it to all change later.

Reply:
Probably a good shout ^